terraform/azure_fxci/local-variables.tf

```diff
@@ -1,61 +1,61 @@
 variable "gecko1" {
-    description = "storage account location"
-    type = map
-    default = {
+  description = "storage account location"
+  type        = map(any)
```

Comment:
This looks like the non-shortform for `map` according to [this link](https://www.terraform.io/language/expressions/type-constraints#map). Is that the case?

Reply:
We can take a look at that. Changes to this file were done by `terraform init` or pre-commit, so we will need to dive in and find out why/how that changed. However, the apply worked without issue only adding the new runbook.